### ISSUE TITLE
pip install not working on windows 

### ISSUE THREAD

**Issue by gracyashhh** [cheofusi/just_playback#8]
```
Building wheel for just-playback (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: 'C:\Users\DELL\AppData\Local\Programs\Python\Python39\python.exe' -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\DELL\\AppData\\Local\\Temp\\pip-install-j_z9gre4\\just-playback_4ad8cf20de5e4d5d993cb87643364567\\setup.py'"'"'; __file__='"'"'C:\\Users\\DELL\\AppData\\Local\\Temp\\pip-install-j_z9gre4\\just-playback_4ad8cf20de5e4d5d993cb87643364567\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d 'C:\Users\DELL\AppData\Local\Temp\pip-wheel-rulb_5ed'
       cwd: C:\Users\DELL\AppData\Local\Temp\pip-install-j_z9gre4\just-playback_4ad8cf20de5e4d5d993cb87643364567\
  Complete output (15 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build\lib.win-amd64-3.9
  creating build\lib.win-amd64-3.9\just_playback
  copying just_playback\ma_result.py -> build\lib.win-amd64-3.9\just_playback
  copying just_playback\playback.py -> build\lib.win-amd64-3.9\just_playback
  copying just_playback\__init__.py -> build\lib.win-amd64-3.9\just_playback
  running build_ext
  generating cffi module 'build\\temp.win-amd64-3.9\\Release\\_ma_playback.c'
  creating build\temp.win-amd64-3.9
  creating build\temp.win-amd64-3.9\Release
  building '_ma_playback' extension
  error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
  ----------------------------------------
  ERROR: Failed building wheel for just-playback
```

The error, can you please help me get rid of this so that I can install the package and use it for my project?

**Comment by cheofusi** [comment]
Upgrade your C++ Build Tools

**Comment by jacklolidk** [comment]
does not work for me.

**Comment by cheofusi** [comment]
Wheels for Windows are  available in [0.1.7](https://pypi.org/project/just-playback/)
